content/en/docs/reference/command-line-tools-reference/kubelet.md

```diff
@@ -579,7 +579,7 @@ ConfigurableFSGroupPolicy=true|false (ALPHA - default=false)<br/>
 CustomCPUCFSQuotaPeriod=true|false (ALPHA - default=false)<br/>
 DefaultPodTopologySpread=true|false (ALPHA - default=false)<br/>
 DevicePlugins=true|false (BETA - default=true)<br/>
-DisableAcceleratorUsageMetrics=true|false (ALPHA - default=false)<br/>
+DisableAcceleratorUsageMetrics=true|false (BETA - default=true)<br/>
```

Comment:
Is this generated documentation? (I didn't check).

Reply:
Unfortunately it seems like it isn't the case: https://github.com/kubernetes/website/blob/7ffb2882dfd0a89233c74cc052c3d4ba7e4bb129/update-imported-docs/reference.yml#L47-L49